python/cudf/cudf/tests/test_reshape.py

```diff
@@ -89,12 +88,7 @@ def test_melt(nulls, num_id_vars, num_value_vars, num_rows, dtype):
     "dtype",
     list(NUMERIC_TYPES + DATETIME_TYPES)
```

Comment:
This is an awkward way to write this. Try `[*NUMERIC_TYPES, *DATETIME_TYPES, "str"]`.

Reply:
Or `list(chain(NUMERIC_TYPES, DATETIME_TYPES, ["str"]))` if you don't like the splats :)

Reply:
used the `chain` method, let me know if there is a strong preference if `chain` isn't a preference.